### PR TITLE
Art create functionality + handler and validator tests

### PR DESCRIPTION
**Pull request by yaninavoda**
## Art gallery block - Create Art
added CreateArtDto
added CreateArtCommand
added validator for CreateArtDto (Title up to 150, Description up to 400, both optional)
added CreateArtHandler (inside this handler don't forget to create StreetcodeArt )
added an action to ArtController
added unit tests for CreateArtHandler
added unit tests for Validator

### CODE REVIEW

**Review by HlibBondarev — APPROVED** [review]
ok

File: Streetcode/Streetcode.BLL/Mapping/Media/Images/ArtProfile.cs [hunk 1]
```diff
@@ -10,5 +9,6 @@ public class ArtProfile : Profile
     public ArtProfile()
     {
         CreateMap<Art, ArtDto>().ReverseMap();
+        CreateMap<Art, CreateArtDto>().ReverseMap();
```
Comment (OlhaPivniuk) [inline Streetcode/Streetcode.BLL/Mapping/Media/Images/ArtProfile.cs:12]: CreateMap<Art, ArtDto>();
CreateMap<CreateArtDto, Art>();

File: Streetcode/Streetcode.BLL/MediatR/Media/Art/Create/CreateArtDtoValidator.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿using FluentValidation;
+using Streetcode.BLL.Dto.Media.Art;
+
+namespace Streetcode.BLL.MediatR.Media.Art.Create;
+
+public class CreateArtDtoValidator : AbstractValidator<CreateArtDto>
+{
+    private readonly int _maxTitleLength;
```
Comment (OlhaPivniuk) [inline Streetcode/Streetcode.BLL/MediatR/Media/Art/Create/CreateArtDtoValidator.cs:8]: use const

File: Streetcode/Streetcode.XUnitTest/ValidationTests/Media/Art/CreateArtValidatorTests.cs [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+    private readonly CreateArtDtoValidator _validator;
+
+    public CreateArtValidatorTests()
+    {
+        _validator = new CreateArtDtoValidator();
+    }
+
+    [Fact]
+    public void Should_not_have_error_when_dto_without_Title()
```
Comment (OlhaPivniuk) [inline Streetcode/Streetcode.XUnitTest/ValidationTests/Media/Art/CreateArtValidatorTests.cs:23]: Rename tests name using previous template name according to our code convension. For example - 
ShouldNotHaveError_WhenDtoWithoutTitle

**Review by OlhaPivniuk — COMMENTED** [review]
Left comments on the code

**Pull request closed without merging**